pom.xml

```diff
@@ -198,6 +199,12 @@
         <version>${version.archunit}</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>slf4j-api</artifactId>
```

Comment:
Do we need to include `slf4j-simple` for tests anywhere? I can see in logs on CI that for `sonar-iac-plugin`
```
[INFO] --- surefire:2.22.2:test (default-test) @ sonar-iac-plugin ---
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
```